dataRetriever/resolvers/topicResolverSender/diffPeerListCreator_test.go

```diff
@@ -13,25 +15,60 @@ import (
 func TestNewDiffPeerListCreator_NilMessengerShouldErr(t *testing.T) {
 	t.Parallel()
 
-	dplc, err := topicResolverSender.NewDiffPeerListCreator(nil, "mainTopic", "excluded")
+	dplc, err := topicResolverSender.NewDiffPeerListCreator(
+		nil,
+		"mainTopic",
```

Comment:
please define constants. "mainTopic" for example is used 8 times in this file.

Reply:
agree, done